cli/exp_task_create.go

```diff
@@ -57,6 +63,26 @@ func (r *RootCmd) taskCreate() *serpent.Command {
 				Description: "Reads from stdin for the task input.",
 				Value:       serpent.BoolOf(&stdin),
 			},
+			{
+				Name:        "wait",
+				Flag:        "wait",
+				Description: "Wait for task completion and stream real-time logs while running.",
+				Value:       serpent.BoolOf(&wait),
+			},
+			{
+				Name:        "wait-timeout",
+				Flag:        "wait-timeout",
+				Description: "How long to wait for task completion.",
+				Value:       serpent.DurationOf(&waitTimeout),
+			},
+			{
+				Name:        "wait-interval",
+				Flag:        "wait-interval",
```

Comment:
```suggestion
				Flag:        "test.wait-interval",
```
Suggestion: highlight test aspect, this pattern is also used elsewhere.